### PR TITLE
Music: remove /projectbeats

### PR DESCRIPTION
**Pull request by breville**
Redirect https://studio.code.org/projectbeats to https://code.org/music.

Also remove the code that supported the long-standing https://studio.code.org/projectbeats page (which proved very useful in incubating this lab).

### CODE REVIEW

File: dashboard/config/routes.rb [hunk 1]
```diff
@@ -41,8 +41,8 @@
     get "/congrats", to: "congrats#index"
 
     get "/incubator", to: "incubator#index"
-    get "/musiclab", to: redirect("/projectbeats", status: 302)
-    get "/projectbeats", to: "musiclab#index"
+    get "/musiclab", to: redirect("/projects/music/new")
```
Comment (cnbrenci) [inline dashboard/config/routes.rb]: Should these go to the tutorial page?

**Review by cnbrenci — APPROVED** [review]
Small Q, but otherwise LGTM!

**Review by molly-moen — APPROVED** [review]
🎉

File: dashboard/config/routes.rb (hunk 1 above)
Comment (breville) [reply dashboard/config/routes.rb]: Good question.  We decided to redirect these to https://code.org/music.

File: apps/src/blockly/TESTING.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ Whenever the mainline `blockly` package version is being upgraded or updates are
 The following is a manual test checklist to follow and is not exhaustive. The estimated time to complete the following checklist is 15-20 minutes.
 
 # For All Mainline Blockly Labs
-Currently, the mainline Blockly labs are: Dance, Poetry, Music (Project Beats), Flappy, Bounce.
+Currently, the mainline Blockly labs are: Dance, Poetry, Music, Flappy, Bounce.
```
Comment (mikeharv) [inline apps/src/blockly/TESTING.md:7]: Thanks for such a thorough clean up! This made me realize we should add Minecraft to this list and replace Poetry with Sprite Lab (its parent lab).

File: dashboard/config/routes.rb [hunk 3]
```diff
@@ -41,8 +41,8 @@
     get "/congrats", to: "congrats#index"
 
     get "/incubator", to: "incubator#index"
-    get "/musiclab", to: redirect("/projectbeats", status: 302)
-    get "/projectbeats", to: "musiclab#index"
+    get "/musiclab", to: redirect(CDO.code_org_url("/music"))
+    get "/projectbeats", to: redirect(CDO.code_org_url("/music"))
```
Comment (mikeharv) [inline dashboard/config/routes.rb:45]: 🎉

**Review by mikeharv — APPROVED** [review]
Nice cleanup!

File: apps/src/blockly/TESTING.md [hunk 4]
```diff
@@ -50,23 +50,23 @@ Create a new Poetry project: [[local_poetry]](http://localhost-studio.code.org:3
...
 - Under 'Effects', check the 'set background to' block. Its dropdown field is grid and includes a 'More' link. When 'More is clicked, modal opens to choose background to add to assets. Note that in July 2023, the 'More' button will change to 'Backgrounds'.
 - Check 'make new_sprite at_' block.
     - Click on the animation dropdown, then click on Costumes button which should take you to Costumes tab.
     - Click on location field, then confirm you can select new location of animation.
 
 
-# Project Beats / Music Lab
-Go to Project Beats: [[local]](http://localhost-studio.code.org:3000/projectbeats) [[production]](https://studio.code.org/projectbeats)
+# Music Lab
+Go to Music Lab: [[local]](http://localhost-studio.code.org:3000/projects/music/new) [[production]](https://studio.code.org/projects/music/new)
```
Comment (sanchitmalhotra126) [inline apps/src/blockly/TESTING.md]: super nit: reword to `"Create a new Music Lab project"` to match the others?

**Review by sanchitmalhotra126 — APPROVED** [review]
👋 farewell /projectbeats!

File: apps/src/blockly/TESTING.md (hunk 4 above)
Comment (breville) [reply apps/src/blockly/TESTING.md]: Good catch.  Done.

**Pull request merged**